label_studio/tests/loadtests/one_imports_other_annotate.py

```diff
@@ -59,19 +60,17 @@ def on_start(self):
 
     @task
     def view_data_manager(self):
-        self.client.get(
-            f'/projects/{self.project_id}/data',
-            name='projects/<pk>/data'
-        )
+        self.client.get(f'/projects/{self.project_id}/data', name='projects/<pk>/data')
 
     # @tag('import')
     # @task
     def import_data(self):
         self.client.post(
             '/api/projects/%i/import' % self.project_id,
             name='/api/projects/<pk>/import',
-            files={'csv': open('data.csv', 'rb')})
-            # headers={'content-type': 'multipart/form-data'})
+            files={'csv': open('data.csv', 'rb')},
+        )
+        # headers={'content-type': 'multipart/form-data'})
```

Comment:
nit: odd it moved this headers comment out of the block it belongs in

Reply:
Blue didn't mind me moving the comment back into the block - I think the move-out was just a side effect of blue rewriting the args list to use a trailing comma.